File: week1/cash/cash.c
#include <stdio.h>
#include <cs50.h>

// variables
int cc = 0; // customer cash
int coins = 0;


// coins: 25, 10, 5, 1

void checkcoins(int cash){
    do{
        if(cash>=25){
            cash-=25;
            coins++;
        }else if(cash>=10){
            cash-=10;
            coins++;
        }else if(cash>=5){
            cash-=5;
            coins++;
        }else if(cash>=1){
            cash-=1;
            coins++;
        }
    }while(cash>0);
}

int main(void)
{
    do{
        cc = get_int("Cash: ");
    }while(cc <= 0);
    checkcoins(cc);
    printf("\n");
    printf("Coins: %d",coins);
    printf("\n");

}

File: week6/cash/cash.py
from cs50 import get_float
cc = 0 # customer cents
cash = 0 # customer cash
coins = 0

while True:
    cash = get_float("Cash: ")
    if cash >= 0:
            break
cc = int(round(cash * 100))
while cc > 0:
        if cc >= 25:
            cc = cc - 25
            coins = coins + 1
        elif cc >= 10:
            cc = cc - 10
            coins = coins + 1
        elif cc >= 5:
            cc = cc - 5
            coins = coins + 1
        elif cc >= 1:
            cc = cc - 1
            coins = coins + 1

print("Coins:", coins)
